keyboards/cannonkeys/savage65/readme.md

```diff
@@ -0,0 +1,12 @@
+# CannonKeys Savage65
+
+Savage65 Keyboard
+
+Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan)  
+Hardware Supported: STM32F072CBT6  
+
+Make example for this keyboard (after setting up your build environment):
+
+    make cannonkeys/savage65:default
+
```

Comment:
something about via support please

Reply:
when its officially in Via, I will add it